### PR TITLE
Fix BatLazyList.remove_all_such documentation

### PR DESCRIPTION
**Pull request by xvw**
Fix a little mistake.

### CODE REVIEW

File: src/batLazyList.mli [hunk 1]
```diff
@@ -420,8 +420,8 @@ val remove_all : 'a -> 'a t -> 'a t
     are equal to [x] and not only the first one. *)
 
 val remove_all_such : ('a -> bool) -> 'a t -> 'a t
-(** [remove_all l x] is similar to [remove] but removes all elements that
-    are equal to [x] and not only the first one. *)
+(** [remove_all_such f l] is similar to [remove] but removes all elements
+    wich satisfay the predicate [f] and not only the first one. *)
```
Comment (gasche) [inline src/batLazyList.mli]: "which satisfy" (or "that satisfy")

**Comment by xvw** [comment]
(Erf, sorry :/, now it's Ok :) )

**Comment by gasche** [comment]
Would you squash the two commits together?

**Comment by xvw** [comment]
(learned a new git jutsu :P)

**Comment by gasche** [comment]
The commit list here is a bit weird, so I suspect the git jitsu didn't completely work.

To squash commits in a pull request, you can use `git rebase -i HEAD~10`, go to the line of the commit to be squashed, and select "squash" (or "s") if you want to amend the commit message or "fixup" ("f") to just merge it to the parent commit without changing the parent commit message (the right choice here). Then use `git push -f your-github-clone-branch` to update the pull request. Note that this works better if you develop each pull request in a separate git branch (to avoid having to do such changes in your master branch).

**Comment by gasche** [comment]
(I merged the patch manually.)

**Pull request closed without merging**